### PR TITLE
[DOC-558] Remove port 9000 from the description of `--yba_url` 

### PR DESCRIPTION
**Pull request by sudhanshu456**
@netlify /preview/yugabyte-platform/configure-yugabyte-platform/on-premises-script/#manually-provision-nodes-using-the-script

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/on-premises-script.md [hunk 1]
```diff
@@ -72,10 +72,10 @@ You can manually provision each node using the pre-provisioning Python script, a
     SUDO password:
     ```
 
-1. Enter your password.
+2. Enter your password.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/on-premises-script.md:75]: You don't need to order the steps as 1,2,3,4. As a markdown convention - https://markdownguide.offshoot.io/basic-syntax/#ordered-lists, we use 1,1,1... as the numbering format. So you can revert the change.

**Review by aishwarya24 — COMMENTED** [review]
Left a minor comment.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/on-premises-script.md (hunk 1 above)
Comment (sudhanshu456) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/on-premises-script.md]: Made the changes
